tests/testthat/test-upkeep.R

```diff
@@ -10,6 +10,8 @@ test_that("tidy upkeep bullets don't change accidentally", {
     )
   )
   create_local_package()
+  # mock test footer so doesn't change with date and usethis version
+  local_mocked_bindings(checklist_footer = function(tidy) "test footer")
```

Comment:
This is one way to do this and it seems fine to me.

You can also use the `transform =` argument of `expect_snapshot()`. If you search "our" code for that, you should be able to surface many examples of what that sort of usage looks like.

Reply:
Oh I like that! I'll do that even if it's just for learning (but I also think it's a bit neater)